appmesh-preview/service-model.json

```diff
@@ -1513,6 +1513,9 @@
                 "validation"
             ],
             "members": {
+                "certificate": {
+                    "shape": "ClientTlsCertificate"
```

Comment:
Does the documentation/description for a field gets added to the GA model only?

Reply:
We haven't been generating API docs for preview features, which is also why the circuit breaking feature is getting its docs in this update. Presumably it's because there isn't a corresponding API docs website for preview which is how people typically consume them. May change in the future.